data/topic-schema/src/main/resources/net/corda/schema/CPI.yaml

```diff
@@ -0,0 +1,22 @@
+topics:
+  CPIInfoTopic:
+    name: cpi.info
+    consumers:
+    producers:

```

Comment:
nit: I'd have preferred to see this introduced only when we had a clear idea of what we're going to do with them

Reply:
I omitted adding any values for this reason. Adding auth requires a change in the prereqs that'll break anyone who's not up to date so will need some announcing